script.js

```diff
@@ -0,0 +1,68 @@
+
+var start_answer = prompt("Hello, sweetie! I'm the Doctor. Doctor, who? Just the Doctor! I'm a Time Lord. Some kind of alien? Well, yes. I've got two hearts and a whole lot of angst. I travel through time and space in my machine called the TARDIS. Will you come with me? (Yes, or No)");
+
+  if (start_answer === "yes") { //YES
+
+
+    var travel_number = prompt("Fantastic! You shall be my companion. I love companions. Especially a sexy one such as yourself. Did I just say that out loud? Well! Let's get on with it then. To when would you prefer to go? \n-the past (1)\n-the present (2)\n-the future (3)");
+
+      if (travel_number === "1") { //PAST
+
+function ElizabethEra(){
```

Comment:
JS functions and variables should be lowercase but camelCase. So instead of `ElizabethEra`, it should be `elizabethEra`.